.github/scripts/requirements.txt

```diff
@@ -3,3 +3,7 @@ pyyaml
 requests
 approvaltests==3.5.0
 pytest
+mypy
+types-PyYAML
+types-requests
+monkeytype
```

Comment:
Can you please mention why we need monkeytype?

Reply:
@argenos Sure:

> Added `add_type_hints.sh` script that uses monkeytype to run code and add missing type hints

It means that if someone adds some new code and doesn't have an IDE that adds types easily, they can run the code and monkeytype will add a starting point for type hints.

Reply:
Is it an optional dependency in that case? Adding it to requirements.txt will install it regardless of whether one uses an IDE or prefers to add the types manually (if I understood right what it does).

Reply:
OK sure. I'll comment out the monkeytype dependency, and add comments in the requirements.txt to divide the groups up and explain them